new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -372,6 +398,14 @@ public static HelmParams installAndVerifyOperator(String opNamespace,
         .domainNamespaces(Arrays.asList(domainNamespace))
         .serviceAccount(opServiceAccount);
 
+    if (domainNamespaceSelectionStrategy != null) {
```

Comment:
you can add/update the existing opParams

```
    if (domainNamespaceSelectionStrategy != null) {
          opParams.domainNamespaceSelectionStrategy(domainNamespaceSelectionStrategy);
    }
```

Reply:
changed